### PR TITLE
App Builders blog

### PR DESCRIPTION
**Pull request by mtnbarreto**

### CODE REVIEW

**Comment by matir91** [comment]
If you want to take a look at it, you can use 
https://blog-staging.xmartlabs.com/2020/04/29/speakers-at-appbuilders-with-covid/

**Comment by bryant1410** [comment]
@matir91 I wonder why it's not available at https://blog-staging.xmartlabs.com/2020/04/29/speakers-at-appbuilders-with-covid/

And if we can do something about this, provided it's related:

image

**Comment by matir91** [comment]
@bryant1410 The post was deployed (you can check [it here](https://github.com/xmartlabs/blog-staging/blob/master/_posts/2020-04-29-speakers-at-appbuilders-with-covid.md)), but it's wrong.

I deployed it locally and jekyll is throwing an error:
```
ekyll_1  |  Incremental build: disabled. Enable with --incremental
jekyll_1  |       Generating... 
jekyll_1  |              Error: YAML Exception reading /srv/jekyll/_posts/2020-04-29-speakers-at-appbuilders-with-covid.md: (<unknown>): could not find expected ':' while scanning a simple key at line 16 column 1
```
I think we should add a "compile" check or something like that in the future.
So my action helped to identify that the post is wrong, :tada:

**Comment by matir91** [comment]
@bryant1410 Now @mtnbarreto fixed the issue, so you can check the blog [here](https://blog-staging.xmartlabs.com/2020/04/29/speakers-at-appbuilders-with-covid/)

File: _posts/2020-05-07-speakers-at-appbuilders-with-covid.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+4. Shorten the talks to make them more dynamic. (Computer attention span seems weaker)
+5. Speaker face must be visible at all times
+6. The ability to create sessions with multiple people for workshops or topic discussions
+
+
+A random small chat picker where people are matched to chat for 2-3 minutes to simulate a real conference.
+We should regard this as motivation to keep working on our local communities, instead of shutting down the sharing of knowledge. We have the tools, all we need is the will to do it.
+
+## Join us at this great event!
+So please come join us at [AppBuilders Conference](https://appbuilders.ch/)! There are a bunch of great speakers like Chris Eidhof (Founder @ objc.io) Peter Steinberger (Founder @ PSPDFKit),Shai Mishali (Creator of RxSwift) and other recognized people from the iOS community coming from big companies like Netflix, Calm, Raywenderlich and [more](https://appbuilders.ch/#speakers). Even Paul Hudson (HackingWithSwift) is chipping in! The event is taking place from May 10th to 12th and my talk is the last day at  [11:45 am (GMT+1)](https://appbuilders.ch/#schedule). See you there! Let's spread the knowledge!
```
Comment (matir91) [inline _posts/2020-05-07-speakers-at-appbuilders-with-covid.md:42]: does (Founder @ objc.io) ok?

**Pull request merged**